en/drafts/originals/text-mining-youtube-comments.md

```diff
@@ -17,115 +17,114 @@ review-ticket: https://github.com/programminghistorian/ph-submissions/issues/374
 
-{% include figure.html filename="en-or-text-mining-youtube-comments-02.png" alt="Screenshot of YouTube video with video ID in browser link circled in red" caption="Figure 2: Screenshot of YouTube video with video ID in browser link circled in red" %}
+There are a variety of resources that can help researchers think through such ethical issues. The University of California at Berkeley hosted a conference on ethical and legal topics in June 2020, recorded in the open access book [_Building Legal Literacies for Text Data Mining_](https://berkeley.pressbooks.pub/buildinglltdm/). Review also [the LLTDM website](https://buildinglltdm.org/), as well as the [Association of Internet Researcher’s Ethics page](https://aoir.org/ethics/) and Annette Markham's [Impact Model for Ethics: Notes from a Talk](https://annettemarkham.com/2017/07/impact-model-ethics/).
 
-For this lesson, we gathered comment data by searching YouTube for “black lives matter george floyd”. We selected a total of six videos from politically polarized news sources (as ranked by allsides.com), including the “left-leaning” sources of New York Times, Vox, and NBC News and the “right-leaning” Daily Mail, Fox News, and the Daily Wire. YouTube makes available a wide range of metadata about each video, including the number of likes, title, description, tags, and more.
+### Video Selection
 
-Choosing multiple videos is often the best approach for the exploratory stages of research, because the YouTube API may not return data for every video searched, even if comment data for that video exists. Wordfish modeling is the most useful, and the most externally valid (meaningful outside the scope of a specific dataset), when the data being modeled is focused on a single topic, but encompasses a variety of viewpoints.
+The most direct way to select videos for research is to visit the [YouTube](https://www.youtube.com/) site, and capture a list of video IDs: these are the set of alphanumeric characters that appear in the video's URL, immediately after `watch?v=`. See for example the video ID circled in red in the illustration below: `24xsqyMcpRg`. These IDs are constant and do not change over time. 
 
-For politically salient topics, an ideal dataset will include an equal number of videos from creators representing both the opposing political perspectives, and should have a substantial number of comments (~2000+) to minimize the skew that outlier comments can introduce. Finally, the total comments from each viewpoint should be balanced. In this case, we made sure that comments from the political ‘left’ and ‘right’ were similar in number. 
+{% include figure.html filename="en-or-text-mining-youtube-comments-02.png" alt="Screenshot of YouTube video with video ID in browser link circled in red" caption="Figure 2. Screenshot of YouTube video with video ID in browser link circled in red" %}

```

Comment:
We'd like to make sure the alt-text for each image in this lesson is more visually descriptive than it is currently: essentially, alt-text should describe the information that we can 'see', in terms that are meaningful to visually-impaired readers.